utils/preformatted-block-names.json

```diff
@@ -19,7 +19,7 @@
   "machine-learning-with-python": "Machine Learning with Python",
   "tensorflow": "TensorFlow",
   "basic-javascript-rpg-game": "Basic JavaScript RPG Game",
-  "basic-html-cat-photo-app": "HTML Cat Photo App",
+  "learn-html-by-building-a-cat-photo-app": "HTML Cat Photo App",
```

Comment:
Why are we not changing the block names?

Reply:
Because the block name didn't match the search 🤦🏻